app/screens/clock_display/clock_display.android.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+    Platform,
```

Comment:
if this file is for android, why would we need `Platform` here?

Reply:
Yup, missed that one